src/util/translation.h

```diff
@@ -22,6 +41,13 @@ struct bilingual_str {
     std::string original;
     std::string translated;
 
+    bilingual_str() = default;
+    bilingual_str(std::string original, std::string translated) : original{original}, translated{translated} {}
+    template <bool translatable>
+    bilingual_str(util::Original<translatable> original) : original{original.lit}, translated{original.translate()}
+    {
+    }
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/31061#discussion_r1793642022

Yes I do think that's better and basically made the same change in my branch, though without the str() method since it seems it makes it a little less clear what return type is.